packages/vscode-dependencies-validation/test/autofix/configuration.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { expect } from "chai";
+import { SinonMock, createSandbox, SinonSandbox } from "sinon";
+import type { WorkspaceConfiguration } from "vscode";
+import {
+  ENABLE_AUTOFIX,
+  getAutoFixDelay,
+  isAutoFixEnabled,
+  internal,
+} from "../../src/autofix/configuration";
+import { VscodeWorkspace } from "../../src/vscodeTypes";
+
+describe("eventUtil unit tests", () => {
+  let sandbox: SinonSandbox;
+  let workspaceConfigurationSinonMock: SinonMock;
+
+  const workspaceConfigurationMock = <WorkspaceConfiguration>{};
+  workspaceConfigurationMock.get = () => "";
+  const workspaceMock = <VscodeWorkspace>{};
+  workspaceMock.getConfiguration = () => workspaceConfigurationMock;
+
+  beforeEach(() => {
+    sandbox = createSandbox();
+    workspaceConfigurationSinonMock = sandbox.mock(workspaceConfigurationMock);
+  });
+
+  afterEach(() => {
+    workspaceConfigurationSinonMock.verify();
+    sandbox.restore();
+  });
+
+  context("isAutoFixEnabled()", () => {
+    it("autofix is disabled", () => {
+      workspaceConfigurationSinonMock
+        .expects("get")
+        .withExactArgs(ENABLE_AUTOFIX, false)
+        .returns(false);
+      const res = isAutoFixEnabled(workspaceMock);
+      expect(res).to.be.false;
+    });
+
+    it("autofix is enabled", () => {
+      workspaceConfigurationSinonMock
+        .expects("get")
+        .withExactArgs(ENABLE_AUTOFIX, false)
+        .returns(true);
+      const res = isAutoFixEnabled(workspaceMock);
```

Comment:
why do we need sinon and assertions on how isAutoFixEnabled implemented to test this?
Can't we just provide a matching placeholder implementation directly to isAutoFixEnabled
and only assert on the final output?

Reply:
I do not think it helps us. Let's talk about it